config/database.yml

```diff
@@ -9,10 +9,14 @@ default: &default
 development:
   <<: *default
   database: shf_project_development
+  username: shf
+  password: shfpassword
```

Comment:
I recommend _not_ using your specific `username` and `password` values for `development` and `test` DB's - as @weedySeaDragon noted, those cannot go into this file, and also doing so just complicates things (and for what purpose?).

Reply:
Thanks for pointing this out. I had trouble connecting to my DB and this was a workaround but I forgot to remove it. Let me work on it